### PR TITLE
versions: image: Update base os to version 20640

### LINKED ISSUE #979: Update image base os to latest CL version
Update Clear Containers image base os to latest version.

### PR DESCRIPTION
**Pull request by jcvenegas**
Update agent to version 867639223bbcd15672626bcd7f58cde2530c0cc5

Changes :

f4fe1d8 network: Revert delete conflicting route
88eb422 CI: Don't require an issue for release PRs
bbd24dc agent: Add support to online vCPUs
ae71318 agent: Fix sync issues with exit code and stdout of a process
16eb82e capabilities: Add support for Linux capabilities
41802c2 agent: add libudev as dependency
ad7c8a5 privileges: Add NoNewPrivileges in the api
4c2b752 devices: Bind mount devices only when /dev is expected to be
d38fb19 dev: Listen to udev events and bind mount these to container's
bd925f1 agent: Make the agent the subreaper of all processes
cf05dda vendor: Update libcontainer vendoring
4dac904 dev: Mount /dev from tmpfs before mount namespace is created
3e12a1b debug: Send errors on the control channel
aa14193 vendor: Update vendoring for virtcontainers
d939c51 network: Use RouteAdd instead of RouteReplace.
9275cec network: Make the error returned on failure to delete a route verbose.
2ec2446 dev: Bind mount /dev/vfio inside container mount namespace
e30849c network: Delete conflicting route

Changes in package coreutils (from 8.28-41 to 8.29-42):
Arjan van de Ven - coreutils: Autospec creation for update from
version 8.28 to version 8.29
https://download.clearlinux.org/releases/19960/clear/RELEASENOTES

Changes in package systemd (from 234-158 to 234-159):
Auke Kok - Allow timesyncd to restart after a
software update.
https://download.clearlinux.org/releases/20480/clear/RELEASENOTES

Changes in package coreutils (from 8.29-42 to 8.29-43):
Victor Rodriguez - coreutils:
    Fix forCVE-2017-18018 CVE-2017-18018-1
Patrick McCarty - Add giturl
https://download.clearlinux.org/releases/20490/clear/RELEASENOTES

Changes in package systemd (from 234-159 to 234-160):
Simental Magana, Marcos - Fix rejected send message by D-Bus from
logind
https://download.clearlinux.org/releases/20620/clear/RELEASENOTES

Fixes #979

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

f26 CI restarted...

**Comment by chavafg** [comment]
K8s failure is because the image couldn't be built:

```
Install Clear containers image
Using /home/fuentess/go/src/github.com/clearcontainers/runtime/versions.txt
version to install cc-20640-agent-867639
download https://github.com/clearcontainers/osbuilder/releases/download/cc-20640-agent-867639/image-cc-20640-agent-867639-binaries.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9    0     9    0     0      8      0 --:--:--  0:00:01 --:--:--     8
100     9    0     9    0     0      8      0 --:--:--  0:00:01 --:--:--     8
package github.com/clearcontainers/packaging: no buildable Go source files in /home/fuentess/go/src/github.com/clearcontainers/packaging
/tmp/cc-bump.h6Lx ~/go/src/github.com/clearcontainers/tests/.ci
Cloning into '/tmp/cc-bump.h6Lx/osbuilder'...
/tmp/cc-bump.h6Lx/osbuilder /tmp/cc-bump.h6Lx ~/go/src/github.com/clearcontainers/tests/.ci
cd scripts; \
docker build  --build-arg GO_VERSION=1.8.3 -t cc-osbuilder .
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.26/build?buildargs=%7B%22GO_VERSION%22%3A%221.8.3%22%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&shmsize=0&t=cc-osbuilder&ulimits=null: dial unix /var/run/docker.sock: connect: permission denied
Makefile:114: recipe for target 'docker-build' failed
make: *** [docker-build] Error 1
[1mINFO[0m: Remove temp dir: /tmp/cc-bump.h6Lx/osbuilder/
```

**Comment by jcvenegas** [comment]
@chavafg seems that ci k8s has not accces to docker socket, I sent https://github.com/clearcontainers/tests/pull/883

**Comment by jcvenegas** [comment]
@chavafg @jodh-intel @sboeuf yes, If I move the agent version I some random issues from cri-o tests. I will collect the errors next time it happen.

**Comment by jcvenegas** [comment]
Now failing because SHASUM file is own by root. 
PR to fix it https://github.com/clearcontainers/tests/pull/886

**Comment by jodh-intel** [comment]
Hi @grahamwhaley - it looks like the metrics CI is failing due to something like https://github.com/clearcontainers/runtime/pull/985 / https://github.com/clearcontainers/tests/pull/877 maybe?

**Comment by grahamwhaley** [comment]
Hi @jodh-intel I need to check exactly why, but at the tail end of last week the metrics CI started failing due to something similar to https://github.com/clearcontainers/osbuilder/issues/8, because our image update/install for CIs moved from downloading binaries to trying to build the latest image via `osbuilder`.
Discussed briefly with @jcvenegas , but I need to check if any of the recent updates have changed the failure case. I didn't think I saw any merges that fixed https://github.com/clearcontainers/osbuilder/issues/8 for us though...

File: .ci/setup.sh [hunk 1]
```diff
@@ -22,7 +22,10 @@
 set -e
 
 test_repo="github.com/clearcontainers/tests"
+packaging_repo="github.com/clearcontainers/packaging"
 
+# Clone packaging repository.
+go get "$test_packaging"
```
Comment (chavafg) [inline .ci/setup.sh]: should we do this on the `tests` repo? If we need it here, we might need it also in the shim, proxy and other repos...

Comment (jodh-intel) [reply .ci/setup.sh]: @jcvenegas - the description suggests this commit might need to be deleted?

**Comment by jcvenegas** [comment]
@grahamwhaley @jodh-intel  pnp jenkins is failing at 
```
08:48:27 + git fetch origin pull/980/head:PR_980
08:48:27 From https://github.com/clearcontainers/runtime
08:48:27  ! [rejected]        refs/pull/980/head -> PR_980  (non-fast-forward)
08:48:28 Build step 'Execute shell' marked build as failure
```
this is because a updated the PR and pushed new  amend my previous commit?
Should we add `--force`  to the git fetch?

I expect after this step is fixed I will fail again at image generation.

**Comment by grahamwhaley** [comment]
If `--force` fixes, that works for me. The other fairly easy alternative (but it will cost us net bandwidth and a bit of build time on the machine) is for me to enable build tree deletion between builds.
Shall we try `--force` first? It also helps with our general paranoia of brute force avoiding any potential niggles that turn up later.

**Comment by jodh-intel** [comment]
@grahamwhaley - if the branches already exist, maybe we should throw in a `git clean -dfx` too?

**Comment by grahamwhaley** [comment]
we could - I see no harm in that (we might want to guard against it failing on the first instance - check the `-e` setting for the relevant script etc.)

**Comment by jcvenegas** [comment]
yay all green!

**Comment by jodh-intel** [comment]
Hi @jcvenegas - can you remove the dnm label now then?

**Pull request merged**